dashboard/app/models/school_district.rb

```diff
@@ -26,6 +26,10 @@ class SchoolDistrict < ApplicationRecord
   #   via http://stackoverflow.com/questions/8073920/importing-csv-quoting-error-is-driving-me-nuts
   CSV_IMPORT_OPTIONS = {col_sep: "\t", headers: true, quote_char: "\x00"}.freeze
 
+  # School district statuses representing currently open districts in 2018-2019 import.
+  # Non-open statuses are 'Closed', 'Future', 'Inactive'
+  OPEN_SCHOOL_STATUSES = ['1-Open', '3-New', '8-Reopened', '5-Changed Boundary/Agency', '4-Added']
```

Comment:
Note I'm waiting on a new file from Liz to use this constant to decide whether a school district still exists or not (we'll populate the `last_known_school_year_open` field accordingly).